clang/test/Driver/clang-s-plugin.s

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang -### -c -fpass-plugin=bar.so -fplugin=bar.so -fplugin-arg-bar-option -Werror %s 2>&1 | FileCheck %s
```

Comment:
This can be moved to `Driver/plugin-driver-args.cpp` with `%clang -### -c -x assembler`

Reply:
Done